temporal-opentracing/src/main/java/io/temporal/opentracing/OpenTracingOptions.java

```diff
@@ -55,13 +60,20 @@ public Tracer getTracer() {
     return tracer;
   }
 
+  @Nonnull
+  public OperationNameAndTagsProvider getOperationNameAndTagsProvider() {
+    return operationNameAndTagsProvider;
+  }
+
   public static Builder newBuilder() {
     return new Builder();
   }
 
   public static final class Builder {
     private Tracer tracer;
     private final Map<SpanOperationType, String> customSpanOperationNamePrefixes = new HashMap<>();
```

Comment:
I would expect this collection to go away with the introduction of ```OperationNameAndTagsProvider```.

I think this collection should be in ```DefaultOperationNameAndTagsProvider``` and somebody could create this ```DefaultOperationNameAndTagsProvider``` with custom profixes map.

Reply:
I agree with this, I was just trying to maintain backwards compatibility

Reply:
are you suggesting that `Builder.setSpanOperationNamePrefix()` would pass through the prefixes to the `DefaultOperationNameAndTagsProvider` instead?

Reply:
No, I think that prefixes should go away from the main `OpenTracingOptions` and `OpenTracingOptionsBuilder` and become a part of the provider itself.
You can create DefaultOperationNameAndTagsProviderOptionsBuilder and make DefaultOperationNameAndTagsProviderOptionsBuilder.setSpanOperationNamePrefix() or provide the map in the DefaultOperationNameAndTagsProvider constructor.

I personally think that breaking of compatibility is alright here, this module was just released and I hope some instability of its API is expected.

Reply:
I removed the old operation name prefix - this is a breaking change, but hopefully nobody will mind too much